.gitignore

```diff
@@ -252,7 +252,6 @@ paket-files/
 *.sln.iml
 
 # MSAL specific ignore files
-docs/
```

Comment:
I agree that having the docs with the library is a big plus. It can be referenced from the .md file, and downloaded with the repo.
I thought I'd see the doc-comments, but I realize they are in already in dev.